drivers/console/Kconfig

```diff
@@ -198,6 +198,15 @@ config XTENSA_SIM_CONSOLE
 	help
 	  Use simulator console to print messages.
 
+config POSIX_SP_CONSOLE
```

Comment:
Can this be made generic instead of being tied to the "single process" board?

Reply:
It is not tied to the simple process board.
If you want to use it on another board you can just modify line 205 to
  depends on BOARD_SIMPLE_PROCESS || BOARD_BANANA

Reply:
I can't imagine this being different for any other POSIX board. Why not simply make it depend on `ARCH_POSIX` instead?